CoordgenFragmenter.cpp

```diff
@@ -144,6 +144,23 @@ void CoordgenFragmenter::initializeInformation(
     orderFragments(fragments, mainFragment);
     molecule->setMainFragment(mainFragment);
     molecule->setFragments(fragments);
+    for_each(fragments.begin(), fragments.end(), setFlipInfo);
+}
+
+bool CoordgenFragmenter::setFlipInfo(sketcherMinimizerFragment* fragment)
```

Comment:
This doesn't need to be exposed in the header, right?

Please make it a static function in the .cpp file.

Reply:
Correct, I did that to stay consistent with all the similar functions -- I just removed each of these from the header as well, along with their docstrings.